lib/fastlane/plugin/maestro_orchestration/helper/maestro_orchestration_helper.rb

```diff
@@ -40,6 +40,52 @@ def self.wait_for_emulator_to_boot(adb, max_retries, serial)
 
         booted
       end
+
+      def self.clear_maestro_logs(clear_logs)
+        return unless clear_logs
```

Comment:
I don't think the helper should have a var whether it executes or not. The helper should also preform the task, the calling code is responsible for whether it calls it or not.

Otherwise if you are a library calling this function you have to add true, which seems odd.